src/Integration.Vsix/SonarLintTagger/TaggerProvider.cs

```diff
@@ -160,15 +163,21 @@ private void OnAnalysisRequested(object sender, AnalysisRequestEventArgs args)
 
             if (detectedLanguages.Any() && analyzerController.IsAnalysisSupported(detectedLanguages))
             {
-                var issueTracker = buffer.Properties.GetOrCreateSingletonProperty(typeof(IIssueTracker),
-                    () => new TextBufferIssueTracker(dte, this, textDocument, detectedLanguages, issuesFilter, sonarErrorDataSource, converter, logger));
+                // We only want one TBIT per buffer and we don't want it be disposed until
```

Comment:
This is exactly the same change that I reverted before. The change was ok, it was just the tests that needed to change.